java/pre_package_concord.bash

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash --login
 set -e # fail fast
 git_root=$(git rev-parse --show-toplevel)
-target_dir=$git_root/jvm/target/concord
-getting_started=$git_root/jvm
+target_dir=$git_root/java/target/concord
```

Comment:
Yea.  I simply renamed jvm to java since we now have 2 jvm languages with the scala example added.